reporting/impl/pom.xml

```diff
@@ -11,6 +11,17 @@
     <artifactId>windup-reporting-impl</artifactId>
     <name>Windup Engine - Reporting Impl</name>
 
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <excludes>
+                    <exclude>**/viz.js</exclude>
```

Comment:
Copy from where?
The thing with JS is that there's no mechanism to prevent JS being duplicated, i.e. some kind of resource dependencies. So if we had, say, viz.js multiple times, it could be in the code several times. Here it is in reporting impl, so kind of general storage for resources... no?